GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeViewModel.swift

```diff
@@ -151,6 +152,7 @@ private extension SecureConversations.WelcomeViewModel {
             case let .success(site):
                 self?.isAttachmentsAvailable = site.allowedFileSenders.visitor
             case .failure:
+                self?.environment.log.prefixed(Self.self).info("Show Unexpected error Dialog")
```

Comment:
would not it be beneficial to log specific error here?

Reply:
https://github.com/salemove/ios-sdk-widgets/pull/852#issuecomment-1854455410